tests/common/integration_features/test_custom_policies_integration.py

```diff
@@ -556,5 +575,45 @@ def mock_custom_policies_response():
     }
 
 
+def mock_multiple_frameworks_custom_policy_response():
+    return {
+        "customPolicies": [
+            {
+                "id": "kpande_bicep_1650378013212",
+                "code": "{\"operator\":\"exists\",\"attribute\":\"spec.runAsUser.rule\",\"cond_type\":\"attribute\","
+                        "\"resource_types\":[\"PodSecurityPolicy\"]}",
+                "title": "bicep policy",
+                "guideline": "meaningful guideline for bicep policy",
+                "severity": "HIGH",
+                "pcSeverity": None,
+                "category": "bicep",
+                "pcPolicyId": None,
+                "additionalPcPolicyIds": None,
+                "sourceIncidentId": None,
+                "benchmarks": {},
```

Comment:
small question: in this example, the first policy should scan only for `bicep` and the second policy scan all frameworks, right?

Reply:
Yes exactly 👍🏻